src/common/mockFactory.ts

```diff
@@ -207,6 +207,54 @@ export default class MockFactory {
         };
     }
 
+    /**
+     * Creates fake IV1Project
+     * @param name Name of project.
+     */
+    public static createTestV1Project(): IV1Project {
+        const regions = MockFactory.createTestV1Regions(10);
+
+        return {
+            frames: {"testFrame.jpg": regions},
```

Comment:
Are we testing importing V1 projects with more than one assets/frames ?

Reply:
Got it! Added multiple assets and multiple regions to the test